### ISSUE TITLE
PSM Folder Visible on AB+C Public Search (on PROD)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#5388]
### Paloma has requested that the PSM permits be visible on the portal public search feature. 

**Notes**
- Use the same display rules as the RW folder.

**Comment by TracyLinder** [comment]
Paloma says however the RW is displaying, PSM should display the same.

**Comment by TracyLinder** [comment]
This has been assigned to LaunchIT.

**Comment by Nadin-Nader** [comment]
Still researching and starting to do change after research of CODES

**Comment by Nadin-Nader** [comment]
Subha has worked on the CODE for this. I have approved the audit in Phabricator for the commit to go through [1ad3804c97c2](http://coaamandadevops01.austintexas.gov/R3:1ad3804c97c2ff12024add176bbbb56b244b64ec) to the sandbox

**Comment by SubhaShroff** [comment]
The code DEV promotion request for commit 1ad3804c97c2, is created on ServiceNow : **SCTASK0129406**

**Comment by Nadin-Nader** [comment]
The Task has been closed and commented with deployed.

**Next Step:**

See if this can go to TEST or if its still on hold @Nadin-Nader

**Comment by Nadin-Nader** [comment]
@SubhaShroff this can be looked at to go to TEST if the UAT is good on your end

**Comment by SubhaShroff** [comment]
@Nadin-Nader  I did a commit -  43f351101fdd. Please approve it.

**Comment by SubhaShroff** [comment]
SCTASK0152427 - DEV deployment request is submitted for commit (43f351101fdd).

**Comment by SubhaShroff** [comment]
@Nadin-Nader  This ticket is looks good on DEV Portal and I recorded the demo video. Demo is sent to you in email for your feedback.

**Comment by Nadin-Nader** [comment]
Forgot to add notes from Fridays meeting.
Per the conversation with Kim at our Friday meeting the "Search by Case Number" filed isnt really used since the PSM folders do not have numbers in the Ref Field in AMANDA.

**Comment by Nadin-Nader** [comment]
@SubhaShroff i have reviewed the demo and it all looks good. Lets get this to TEST


## APPROVED for TEST Deployment‼🎉🙌

**Comment by SubhaShroff** [comment]
@Nadin-Nader  TEST deployment request is submitted - **SCTASK0152966**.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  TEST deployment is done. I tested it and its ready for your UAT.

**Comment by SubhaShroff** [comment]
### Status:
Up for UAT on TEST.

**Comment by Nadin-Nader** [comment]
@SubhaShroff i have UATd this on TEST and it looks like its working properly.

**Comment by Nadin-Nader** [comment]
## **Status:**

## **Ready for TEST UAT🙌🙌**

**Comment by Nadin-Nader** [comment]
Sent email to stakeholders informing them that this is ready for TEST UAT:

email to Paloma, Kim and Isabel:

> Good Evening,
> 
> Hope you are all doing well. I was reaching out again tonight to inform you that the request to have the PSM Folder Visible on Portal # 5388 is ready for you to UAT on TEST. 
> Please go ahead and UAT this on TEST and let me know if it is good to go to PROD. Once approved we will get this over to the DevOps CM team to put on PROD.
> 
> Respectfully,
> 
> Nadin A. Nader (He/Him/His)
> IT Business Systems Analyst
> Data and Technology Services
> ————————————————————————————
> transportation.data@austintexas.gov 🚦 austinmobility.io

**Comment by Nadin-Nader** [comment]
## **Status:**

**Waiting on Stakeholder UAT results on TEST**

**Comment by Nadin-Nader** [comment]
@SubhaShroff this has been approved to go to PROD

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0157747 is submitted for PROD promotion.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  This ticket is promoted to PROD.

**Comment by SubhaShroff** [comment]
### STatus:
Promoted to PROD.

**Comment by SubhaShroff** [comment]
### Status:
Promoted to PROD in the last week.

**Comment by Nadin-Nader** [comment]
This has been approved by Paloma on PROD. Closing
